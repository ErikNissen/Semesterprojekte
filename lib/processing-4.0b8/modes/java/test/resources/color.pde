version https://git-lfs.github.com/spec/v1
oid sha256:cd7c0452706ac36a5db675e986efd4adfd8a70da2b98be7d310cdbe12d2657f0
size 92
