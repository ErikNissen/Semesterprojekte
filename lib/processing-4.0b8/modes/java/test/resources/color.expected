version https://git-lfs.github.com/spec/v1
oid sha256:d0bd7d906dc160cf0dc16342cdaa50c84e1c77f0f951b11f9bbca3fe30241988
size 778
